### ISSUE TITLE
Deprecated! Please use global 'renovate' command. && Config needs migrating

### ISSUE THREAD

**Issue by Albertbol** [renovatebot/config-help#901]
### Which Renovate are you using?

Renovate Open Source CLI

### Which platform are you using?

GitLab.com

### Have you checked the logs? Don't forget to include them if relevant

Gitlab pipeline log:
$ ln -sF $CI_PROJECT_DIR/config.js /usr/src/app/
$ node /usr/src/app/dist/renovate.js
 WARN: Deprecated! Please use global 'renovate' command.
 WARN: Config needs migrating

### What would you like to do?
I would like to ask about 2 WARN messages in GitLab pipeline:
` WARN: Deprecated! Please use global 'renovate' command.`
And
` WARN: Config needs migrating`

I could not find info about solving those 2 issues.

Here is my `.gitlab-ci.yml`

```
renovate:
  variables:
    RENOVATE_TOKEN: $GITLAB_ACCESS_TOKEN
    GITHUB_COM_TOKEN: $GITHUB_ACCESS_TOKEN
    DOCKER_USERNAME: $CI_REGISTRY_USER # needed to allow Renovate to check against GitLabs own Docker container registry
    DOCKER_PASSWORD: $CI_REGISTRY_PASSWORD
  image: registry.gitlab.com/***/container-registry:renovate
  script:
    - ln -sF $CI_PROJECT_DIR/config.js /usr/src/app/
    - node /usr/src/app/dist/renovate.js
  only:
    - schedules
```

`config.js`:
```
module.exports = {
    platform: 'gitlab',
    baseBranches: ['master'],
    labels: ['renovate'],
    rebaseWhen: 'auto',
    semanticCommits: true,
    major: {
      automerge: false
    },
    repositories: [
      '*****'
    ],
    extends: [
      'config:base',
      ":pinAllExceptPeerDependencies"
    ]
  };
```
`renovate.json`:
```
{
    "$schema": "https://docs.renovatebot.com/renovate-schema.json"
  }
```

**Comment by rarkins** [comment]
The config warning is self explanatory and you should be able to find the before and after migrating configs in the logs.

**Comment by rarkins** [comment]
There should be additional logs before or after the config migration warning. Please log at debug level and look for them.

**Comment by rarkins** [comment]
As you are not using the official image then you probably need to debug the first problem yourself.

**Comment by viceice** [comment]
You should use the global renovate executable.

So instead `node ...` run `renovate`.

**Comment by Albertbol** [comment]
First of all, wanted to say thanks for a great job and a great library.
1. I changed:
`node /usr/src/app/dist/renovate.js` to `renovate` and it fixed an issue with deprecation.
2. `Config needs migrating` message even without `logLevel: 'debug' `line  in `config.js` outputs originalConfig and migratedConfig, I compared both and saw conflict:
I had:
`"semanticCommits": true`
And migrated config had:
`"semanticCommits": "enabled"`
Read about it in [official docs](https://docs.renovatebot.com/configuration-options/#semanticcommits) and just removed it, so it will be autodetected. It helped.

Thanks a log again for such a fast help on Sunday + for a great library again ;)
